### DISCUSSION TITLE
Batch antivirus

### DISCUSSION THREAD

**Discussion by CyberSec123123** [anic17/Batch-Antivirus#12] [General]
I think that one of the GitHub batch antivirus is nice enough by anic17. So problem is that batch file is attached with three exe files. And this is very disgusting me. We need entire open source code. Please fix this.

**Comment by anic17** [comment]
> I think that one of the GitHub batch antivirus is nice enough by anic17. So problem is that batch file is attached with three exe files. And this is very disgusting me. We need entire open source code. Please fix this.

Sorry for the inconveniences, I will try to find the source of all executables.

**Comment by anic17** [comment]
I uploaded the source code of the program `gethex.exe` on [`source`](https://github.com/anic17/Batch-Antivirus/blob/master/source/gethex.c) folder and [`digest.c`](https://github.com/anic17/Batch-Antivirus/blob/master/source/digest.c) for `sha256.exe`.  

Unfortunately, I wasn't able to find the source code of `waitditchange.exe`, as the place where I downloaded (TheBATeam repository) only included the executable.

Thanks for bringing this into attention,
anic17
